### PR TITLE
fix: add noHtml to axe.configure

### PR DESCRIPTION
**Pull request by straker**

### CODE REVIEW

File: lib/core/utils/dq-element.js [hunk 1]
```diff
@@ -39,8 +39,12 @@ function DqElement(element, options, spec) {
...
    * @type {String}
    */
-  this.source =
-    this.spec.source !== undefined ? this.spec.source : getSource(element);
+  // TODO: es-moduels_audit
+  this.source = axe._audit.noHtml
+    ? ''
+    : this.spec.source !== undefined
+    ? this.spec.source
+    : getSource(element);
```
Comment (WilcoFiers) [inline lib/core/utils/dq-element.js:47]:
```suggestion
  if (axe._audit.noHtml) {
    this.source = ''
  } else if (this.spec.source !== undefined) {
    this.source = this.spec.source
  } else {
    this.source = getSource(element)
  }
```

File: lib/core/utils/dq-element.js [hunk 2]
```diff
@@ -39,8 +39,12 @@ function DqElement(element, options, spec) {
    * The generated HTML source code of the element
    * @type {String}
    */
-  this.source =
-    this.spec.source !== undefined ? this.spec.source : getSource(element);
+  // TODO: es-moduels_audit
+  this.source = axe._audit.noHtml
+    ? ''
```
Comment (WilcoFiers) [inline lib/core/utils/dq-element.js:44]: Should this be undefined?

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
I would like some integration tests that prove that 1. this does not send HTML over postMessage, and 2. it does not return html, either from the page level or from frames.

File: lib/core/utils/dq-element.js [hunk 3]
```diff
@@ -39,8 +39,14 @@ function DqElement(element, options, spec) {
    * The generated HTML source code of the element
    * @type {String}
    */
-  this.source =
-    this.spec.source !== undefined ? this.spec.source : getSource(element);
+  // TODO: es-moduels_audit
```
Comment (stephenmathieson) [inline lib/core/utils/dq-element.js:42]: Typo

```suggestion
  // TODO: es-modules_audit
```

File: lib/core/utils/dq-element.js (hunk 2 above)
Comment (stephenmathieson) [reply lib/core/utils/dq-element.js]: `undefined` or `null`? IMO `null` is preferred, since it's explicitly not set. NBD tho :)

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
For the record; this does add a small feature, but it is a feature necessary to address a security fix that will need to be applied to all supported minor versions of axe-core. We are therefor tagging this as a "fix" instead of a "feat".

**Pull request merged**